### PR TITLE
Bootstrap and navbar added

### PR DESCRIPTION
**Pull request by douvel**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -2,11 +2,16 @@
...
+  <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
   <%= stylesheet_link_tag    'application', media: 'all', 'data-turbolinks-track' => true %>
   <%= javascript_include_tag 'application', 'data-turbolinks-track' => true %>
   <%= csrf_meta_tags %>
 </head>
 <body>
+
+<%= render 'shared/navbar' %>
+
   <p class="notice"><%= notice %></p>
```
Comment (jowarni) [inline app/views/layouts/application.html.erb]: Can you bootstrapize these flashes?

**Comment by jowarni** [comment]
You should also remove `stylesheets.zip`

**Comment by jowarni** [comment]
Review done.

**Comment by douvel** [comment]
Fix done

**Pull request merged**